tools/sslsniff_example.txt

```diff
@@ -160,12 +160,18 @@ READ/RECV    0.000049464        nginx            7081    1      0.004
 
 ----- END DATA -----
 
+Tracing few extra libraries (useful for docker containers and other isolated
+apps)
+
+# ./sslsniff.py --extra-lib /var/lib/docker/overlay2/l/S4EMHE/lib/libssl.so.1.1
```

Comment:
openssl:/var/lib/docker/overlay2/l/S4EMHE/lib/libssl.so.1.1?

Reply:
Yeah you right, don't even know how I missed this :shrug: .